src/index.ts

```diff
@@ -19,11 +19,11 @@ declare global {
 
 export const injectAxe = async (page: Page): Promise<void> => {
   const axe: string = fs.readFileSync(
-    'node_modules/axe-core/axe.min.js',
-    'utf8',
-  )
-  await page.evaluate((axe: string) => window.eval(axe), axe)
-}
+    `${require.resolve("axe-core/axe.min.js")}`,
+    "utf8"
+  );
+  await page.evaluate((axe: string) => window.eval(axe), axe);
+};
```

Comment:
```suggestion
    require.resolve('axe-core/axe.min.js'),
    'utf8',
  )
  await page.evaluate((axe: string) => window.eval(axe), axe)
}
```

Reply:
I think this will fail the codacy job. Probably needs a bit more tuning.